admin/common_auth/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from django.contrib.auth import authenticate
+
+class LoginForm(forms.Form):
+    username = forms.CharField(label=(u'User Name'))
+    password = forms.CharField(label=(u'Password'), widget=forms.PasswordInput(render_value=False))
+
+    def clean(self):
```

Comment:
You should be calling authenticate in the view (which you are also doing), not in the clean. You can actually remove the clean method entirely if you add `required=True` as kwargs to both of the fields. Because it inherits from `forms.Form` it will automatically call `.clean()` on the parent class.

[An example of calling authenticate and then login in the view.](https://docs.djangoproject.com/en/1.8/topics/auth/default/#django.contrib.auth.login)

[You can then use the messages framework in the view to tell them they've entered incorrect login information.](https://docs.djangoproject.com/en/1.8/ref/contrib/messages/#django.contrib.messages.add_message) and then [redirect them](https://docs.djangoproject.com/en/1.8/topics/http/shortcuts/#redirect) back to the login form.